azurerm/internal/services/policy/tests/policy_set_definition_resource_test.go

```diff
@@ -619,6 +654,35 @@ PARAMETERS
 `, data.RandomInteger, data.RandomInteger)
 }
 
+func testAzureRMPolicySetDefinition_templateNoParameter(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_policy_definition" "test" {
+  name         = "acctestpol-%d"
+  policy_type  = "Custom"
+  mode         = "All"
+  display_name = "acctestpol-%d"
+
+  policy_rule = <<POLICY_RULE
+	{
+    "if": {
+      "not": {
+        "field": "location",
+        "equals": "eastus"
```

Comment:
this won't work in regions other than Azure Public - so we need to pass this in here